src/services/mockDb.js

```diff
@@ -65,6 +65,64 @@ const mockdb = {
       }
     }
     if (qstring === 'INSERT INTO users (username, password) VALUES (?, ?);') callback(null, { insertId: 1 });
+    // GET hero/:heroId

```

Comment:
remove unnecessary comment

Reply:
Every section has a comment in this file, marking which endpoint they relate to, so I followed the already established and merged pattern. Would you like me to remove the other comments as well?

Reply:
yes please, thank you!

Reply:
Sorted!